### PR TITLE
SEO-2852-ASP.NET-MVC-Switch-Int

### PR DESCRIPTION
**Pull request by berylouma**
Hi @ChristoIssac

|Title|Description|
|--|--|
|Task Category|ASP.NET MVC Toggle Switch Button Money Keywords|
|Content Task Link/Mail Screenshot|https://syncfusion.atlassian.net/browse/CONTENT-7463|
|UX task|NA|
|Master / Dev / Inter PR link|Inter - https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1123|
|Ticket/Task link|https://syncfusion.atlassian.net/browse/SEO-2852
|Excel/SharePoint link| Retained Keywords, volume, and position. https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/ASP.NET-MVC-Toggle-Switch-Button---Money-Keywords.aspx|

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|asp.net mvc toggle switch button  |0-10 |3FT|Improve FT position|Made on-page changes|
|asp.net mvc switch button|10-100| 2FT|Improve FT position|Made on-page changes|
|asp.net switch button|10-100 |1UG |To work on FT positioning|Made on-page changes|
|asp.net mvc toggle button control|0-10 |1FT |To maintain FT position|Made on-page changes|

Worked on 15 keywords in the react dialog and am keeping track of all keyword details in the link below:-https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/Keyword-Plan---ASP.NET-MVC-Toggle-Switch-Button-Money-Keywords.aspx
Regards,
Beryl.

### CODE REVIEW

File: ej2-asp-core-mvc/switch/EJ2_ASP.MVC/getting-started.md [hunk 1]
```diff
@@ -149,4 +149,6 @@ This section explains how to set [OnLabel](https://help.syncfusion.com/cr/aspnet
 
 ## See also
 
-* [How to customize the switch appearance](./how-to/customize-the-appearance-of-a-switch)
\ No newline at end of file
+* [How to customize the switch appearance](./how-to/customize-the-appearance-of-a-switch)
+
```
Comment (Girija-Rajendran) [inline ej2-asp-core-mvc/switch/EJ2_ASP.MVC/getting-started.md:153]: image

Comment (berylouma) [reply ej2-asp-core-mvc/switch/EJ2_ASP.MVC/getting-started.md:153]: Changes made

**Review by Girija-Rajendran — APPROVED** [review]

File: ej2-asp-core-mvc/switch/EJ2_ASP.MVC/getting-started.md [hunk 2]
```diff
@@ -149,4 +149,6 @@ This section explains how to set [OnLabel](https://help.syncfusion.com/cr/aspnet
...
+N> You can refer to our [ASP.NET MVC Toggle Switch Button](https://www.syncfusion.com/aspnet-mvc-ui-controls/toggle-switch-button) feature tour page for its groundbreaking feature representations. You can also explore our [ASP.NET MVC Toggle Switch Button example](https://ej2.syncfusion.com/aspnetmvc/Button/Switch#/material) that shows you how to render the Toggle Switch Button in ASP.NET MVC.
```
Comment (Sharly-Atieno) [inline ej2-asp-core-mvc/switch/EJ2_ASP.MVC/getting-started.md:154]: update correct demo link

Comment (berylouma) [reply ej2-asp-core-mvc/switch/EJ2_ASP.MVC/getting-started.md]: Updated

**Pull request closed without merging**